src/Build/BuildCheck/API/IBuildCheckRegistrationContext.cs

```diff
@@ -9,8 +9,11 @@ public interface IBuildCheckRegistrationContext
 {
     void RegisterEvaluatedPropertiesAction(Action<BuildCheckDataContext<EvaluatedPropertiesCheckData>> evaluatedPropertiesAction);
 
+    [Obsolete("Use RegisterEvaluatedItemsAction to obtain evaluated items of a project.", false)]
```

Comment:
I think we should add to the message when we plan to deprecate it? Both for users to be aware and as a reminder for us to clean up the code.

Reply:
Let's stay with a tracking bug - we cannot (and actually should not) publicly promisse any timelines